src/seedu/addressbook/data/person/Address.java

```diff
@@ -9,10 +9,19 @@
 public class Address {
 
     public static final String EXAMPLE = "123, some street";
-    public static final String MESSAGE_ADDRESS_CONSTRAINTS = "Person addresses can be in any format";
+    public static final String MESSAGE_ADDRESS_CONSTRAINTS = "Person addresses must be entered in this format: a/BLOCK, STREET, UNIT, POSTAL_CODE";
     public static final String ADDRESS_VALIDATION_REGEX = ".+";
 
+    public static final int INDEX_OF_BLOCK = 0;
+    public static final int INDEX_OF_STREET = 1;
+    public static final int INDEX_OF_UNIT = 2;
+    public static final int INDEX_OF_POSTALCODE = 3;
+    
     public final String value;
```

Comment:
What is the point of 'value' variable now that you have separate classes to maintain the components in the address?